test/RibbonThetaVaultYearn.js

```diff
@@ -3426,6 +3408,104 @@ function behavesLikeRibbonOptionsVault(params) {
       });
     });
 
+    describe("#sunset", () => {
+      time.revertToSnapshotAfterEach();
+
+      it("reverts when not owner", async function () {
+        await expect(
+          this.vault.sunset(this.v2vault.address)
+        ).to.be.revertedWith("Ownable: caller is not the owner");
+      });
+
+      it("succeeds when owner", async function () {
+        await this.vault.connect(ownerSigner).sunset(this.v2vault.address);
+
+        assert.equal(await this.vault.replacementVault(), this.v2vault.address);
+      });
+
+      it("succeeds in resetting replacementVault", async function () {
+        await this.vault.connect(ownerSigner).sunset(owner);
+        await this.vault.connect(ownerSigner).sunset(this.v2vault.address);
+
+        assert.equal(await this.vault.replacementVault(), this.v2vault.address);
+      });
+
+      it("deposits still work after sunset", async function () {
+        await this.vault.connect(ownerSigner).sunset(this.v2vault.address);
+        const depositAmount = BigNumber.from("10000000000");
+        await depositIntoVault(params.depositAsset, this.vault, depositAmount);
```

Comment:
This will deposit into a v2 vault correct?

Reply:
v1 vault, just testing the that depositing works after sunset